command_test.go

```diff
@@ -377,3 +377,26 @@ func TestCommand_Run_CustomShellCompleteAcceptsMalformedFlags(t *testing.T) {
 	}
 
 }
+
+func TestCommand_NoVersionFlagOnCommands(t *testing.T) {
+	app := &App{
+		Version: "some version",
+		Commands: []*Command{
+			{
+				Name:        "bar",
+				Usage:       "this is for testing",
+				Subcommands: []*Command{{}}, // some subcommand
+				HideHelp:    true,
+				Action: func(c *Context) error {
+					if len(c.App.VisibleFlags()) != 0 {
+						t.Fatalf("unexpected flag on command")
```

Comment:
Nit: Since there's no formatting here, `t.Fatal` is more appropriate.

Reply:
fixed